package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "foreman_discovery",
+  "version": "1.0.0",
+  "license": "GPL-3.0",
+  "description": "Node stack for foreman-discovery",
+  "scripts": {},
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/theforeman/foreman_discovery.git"
+  },
+  "devDependencies": {
+    "babel-eslint": "^8.2.2",
+    "babel-plugin-lodash": "^3.3.2",
+    "babel-plugin-transform-class-properties": "^6.24.1",
+    "babel-plugin-transform-object-assign": "^6.22.0",
+    "babel-plugin-transform-object-rest-spread": "^6.26.0",
+    "babel-preset-env": "^1.6.1",
+    "babel-preset-react": "^6.24.1",
+    "css-loader": "^0.28.11",
+    "eslint": "^4.19.1",
+    "eslint-config-airbnb-base": "^12.1.0",
+    "eslint-plugin-import": "^2.10.0",
+    "eslint-plugin-jest": "^21.15.0",
+    "eslint-plugin-jsx-a11y": "^6.0.3",
+    "eslint-plugin-react": "^7.7.0",
+    "jest": "^22.4.3",
+    "jest-cli": "^22.4.3",
+    "stylelint": "^9.2.0",
+    "stylelint-config-standard": "^18.2.0",
+    "uglifyjs-webpack-plugin": "^1.2.4"
+  },
+  "dependencies": {
+    "babel-polyfill": "^6.26.0",
```

Comment:
I would suggest not to load these unless you are adding code that uses it.

Reply:
I wish to start moving stuff slowly to react, that's why I'ved added them, at the moment I want to start with supporting of scss loading using `.js` file.